src/shared/common/forms/composite/entity-list.jsx

```diff
@@ -96,6 +97,7 @@ class EntityList extends Component {
                             })}
                             <br />
                             {hasAddLink && <Link to={`/case/${page.params.caseId}/stage/${page.params.stageId}/entity/${entity}/add?hideSidebar=${hideSidebar}`} className="govuk-body govuk-link">Add a {entity}</Link>}
+                            {hasAddLinkNoMp && <Link to={`/case/${page.params.caseId}/stage/${page.params.stageId}/entity/${entity}/addNoMp?hideSidebar=${hideSidebar}`} className="govuk-body govuk-link">Add a {entity}</Link>}
```

Comment:
I wonder if you genericise this like the `sumo-list`. So have a primaryLink item instead that changes the where the action should point to? 

`{hasAddLinkNoMp && <Link to={`/case/${page.params.caseId}/stage/${page.params.stageId}/entity/${entity}/addNoMp?hideSidebar=${hideSidebar}`} className="govuk-body govuk-link">Add a {entity}</Link>}` 
would then become something similar to 
`{primaryLink && <Link to={`/case/${page.params.caseId}/stage/${page.params.stageId}/entity/${entity}/{primaryLink.action}?hideSidebar=${hideSidebar}`} className="govuk-body govuk-link">Add a {entity}</Link>}`

Would be a little work but it would be a copy from `somu-list` pretty much.